rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/scan/ast/JavaTypeReferenceModel.java

```diff
@@ -18,8 +18,21 @@
 
     public static final String TYPE = "JavaTypeReference";
     public static final String REFERENCE_TYPE = "referenceType";
+    public static final String RESOLVED_SOURCE_SNIPPIT = "resolvedSourceSnippit";
 
     /**
+     * Gets the snippit referenced by this {@link FileLocationModel}.
+     */
+    @Property(RESOLVED_SOURCE_SNIPPIT)
+    void setResolvedSourceSnippit(String source);
```

Comment:
I think that I would prefer this field be used for the unresolved source snippit. I'm not sure why we need to change the default behavior for referencedSourceSnippit?